app/jobs/update_job.rb

```diff
@@ -55,6 +55,17 @@ def perform(model_params:, signed_ids:, background_job_result:, start_workflow:
     end
 
     background_job_result.complete!
+  rescue Dor::Services::Client::BadRequestError => e
+    # report as error and do not retry
+    background_job_result.output = { errors: [title: 'HTTP 400 (Bad Request) from dor-services-app',
```

Comment:
might want to do the `background_job_result.output.merge` like in the `rescue StandardError` case, so as to keep the `{ druid: model.externalIdentifier }` from line 42?  or, will this always come from `object_client.update` on line 40, implying that `background_job_result.output` will never be set when this error is hit?

Reply:
good point, fixed.